services/m3aggregator/config/aggregator.go

```diff
@@ -436,7 +461,11 @@ func (c placementManagerConfiguration) NewPlacementManager(
 
 type forwardingConfiguration struct {
 	// MaxSingleDelay is the maximum delay for a single forward step.
-	MaxSingleDelay time.Duration `yaml:"maxSingleDelay"`
+	MaxSingleDelay         time.Duration                      `yaml:"maxSingleDelay"`
```

Comment:
if this is required, add a yaml validation?

Reply:
It's not required, if unspecfied, it uses the default value in the options, I'll make it a `*time.Duration`.